src/System.Text.Json/tests/Utf8JsonReaderTests.TryGet.cs

```diff
@@ -263,9 +290,202 @@ public static void TestingNumbersInvalidConversionToInt64(string jsonString, dou
             {
                 if (json.TokenType == JsonTokenType.Number)
                 {
-                    Assert.False(json.TryGetInt64Value(out long value));
-                    Assert.True(json.TryGetDoubleValue(out double doubleValue));
+                    #region TryGet calls
+                    Assert.False(json.TryGetInt64(out long value));
+                    Assert.True(json.TryGetDouble(out double doubleValue));
                     Assert.Equal(expected, doubleValue);
+                    #endregion
+
+                    #region Get calls
+                    try
+                    {
+                        json.GetInt64();
+                        Assert.True(false, "Expected GetInt64 to throw FormatException.");
+                    }
+                    catch (FormatException)
+                    {
+                        /* Expected exception */
+                    }
+                    doubleValue = json.GetDouble();
+                    Assert.Equal(expected, doubleValue);
+                    #endregion
+                }
+            }
+
+            Assert.Equal(dataUtf8.Length, json.BytesConsumed);
+            Assert.Equal(json.BytesConsumed, json.CurrentState.BytesConsumed);
+        }
+
+        [Theory]
+        [InlineData("0.000", 0)]
+        [InlineData("1e1", 10)]
+        [InlineData("1.1e2", 110)]
+        [InlineData("12345.1", 12345.1)]
+        [InlineData("12345678901", 12345678901)]
+        [InlineData("123456789012345678901", 123456789012345678901d)]
+        [InlineData("-1", -1)] // uint.MinValue - 1
+        public static void TestingNumbersInvalidConversionToUInt32(string jsonString, double expected)
+        {
+            byte[] dataUtf8 = Encoding.UTF8.GetBytes(jsonString);
+
+            var json = new Utf8JsonReader(dataUtf8, true, default);
```

Comment:
nit: specify the optional parameter name:
`var json = new Utf8JsonReader(dataUtf8, isFinalBlock: true, state: default);`

Reply:
Yep, I realized that you didn't write it like this, but while reviewing thought I would mention it. Just a nit, feel free to ignore :)